userguide/intro.rst

```diff
@@ -333,6 +333,10 @@ These screen options have changed:
 * iocage jails can now be restarted from the |web-ui| in
   :menuselection:`Jails -->` |ui-options| :menuselection:`--> Restart`.
 
+* The :guilabel:`Hostname` and :guilabel:`Domain` set in
+  :menuselection:`Network --> Global Configuration`
+  is now always displayed under the iXsystems logo.
```

Comment:
"always" has a strange sound here, not necessary.  Should tell where that is, an end user reading this might not realize where the logo is on the screen.